whylog/constraints/__init__.py

```diff
@@ -9,12 +10,35 @@ class AbstractConstraint(object):
 
 
 class TimeConstraint(AbstractConstraint):
+    MIN_DELTA = 'min_delta'
+    MAX_DELTA = 'max_delta'
+
     def __init__(self, groups, param_dict):
         pass
 
     @classmethod
     def verify(cls, group_contents, param_dict):
-        # TODO remove mock
+        """
+        :param group_contents
+        group_contents[0] = date supposed to be earlier (cause)
+        group_contents[1] = date supposed to be later (effect)
+        :param param_dict
+        contains 'min_delta' and 'max_delta', time interval between
+        effect and cause should satisfy requirement:
+        min_delta < time interval < max_delta
```

Comment:
Maybe `time delta` instead of `time interval` it would be more consistent with variable naming in following implementation